src/datreant/core/collections.py

```diff
@@ -148,26 +148,29 @@ def __getitem__(self, index):
         select out members.
 
         """
+        memberlist = self._list()
         # we can take lists of indices, names, or uuids; these return a
         # View; repeats already not respected since View functions as a
         # set
-        if ((isinstance(index, list) or hasattr(index, 'dtype')) and
-                all([isinstance(item, bool) for item in index])):
```

Comment:
The issue says not to import numpy, I imagine so this package can work without it and remains pure python (incl. dependencies)?  Seemed like a fun coding challenge either way :)